src/components/RegistrationSuccessPage/RegistrationSuccessPage.jsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+import styles from "./LoginSucccessPage.scss";
+import Envelope from "./img/envelope.png";
```

Comment:
I don't see usage of this three imports Envelope, Logo, Footer
If you don't use them, just remove